.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,18 +1,35 @@
-<!--  Thanks for sending a pull request!  Here are some tips for you:
```

Comment:
in your commit message, please explain why your suggested template is better than the former one.

Frankly, I'd *add* this request to the template. Every PR should give a reasoning on its motivation.

Reply:
I see too much changes in this PR for a single readme file. I'll close this PR and open another in the future.